src/docs/faqs/faqs.md

```diff
@@ -89,44 +96,52 @@ enter „cmd“ and execute the command there, as an alternative, navigate in th
 
 In case you want to delete the Test repo again: navigate into the project within the DataHUB, on the left hand side select settings and general
 ->advanced->Expand-> at the bottom: “ Delete project”.
+</details>
+<br>-->
 
-## Swate
-
-**Q:**  Is it possible to use common Excel functionalities?
-
-
-**A:**  Yes! In fact, this is one of the reasons why we decided to embed Swate within Excel, as it allows users to continue using the Excel functionality they are accustomed to. 
 
-**Q:**  Do I have to fill in all columns when using a template?
-
-**A:**  
-
-**Q:**  Which format must be used to enter a date?
-
-**A:**  String formatted as ISO8601 date
-YYYY-MM-DD
-
-ISA-Spec
+## Swate
 
-**Q:**  Is it necessary to fill in the empty fields in the investigation-file manually? 
+<details><summary><span style="color: #1fc2a7;font-size:1.2em">
+Is it possible to use common Excel functionalities?
+</summary>
+Yes! In fact, this is one of the reasons why we decided to embed Swate within Excel, as it allows users to continue using the Excel functionality they are accustomed to.
+</details>
 
-**A:** Eigentlich muss man da gar nichts amchen, aber wenn man will kann man, aber generell gilt: Was nicht passt, einfach frei lassen
+<details><summary><span style="color: #1fc2a7;font-size:1.2em">
+Do I have to fill in all columns when using a template?
+</summary>
+No! Except you have used the template to reach a specific endpoint repository. Here, we have taken care to query only the necessary metadata in the corresponding templates.
```

Comment:
"No. However, if you want to submit your data to a [public data repository](https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/fundamentals/PublicDataRepositories.html), these will require specific metadata. For this purpose we provide the corresponding templates helping you to annotate your data accordingly."